<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="okno" name="obraz" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="alpha">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>340</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="obraz1">
       <property name="text">
        <string>Warstwa1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="obraz2">
       <property name="text">
        <string>Warstwa2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="obraz3">
       <property name="text">
        <string>Warstwa3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>140</y>
      <width>160</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="dodawanie2">
       <property name="text">
        <string>Suma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="odejmowanie2">
       <property name="text">
        <string>Roznica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mnozenie2">
       <property name="text">
        <string>Iloczyn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lighten2">
       <property name="text">
        <string>Lighten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="difference2">
       <property name="text">
        <string>Difference</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>okno</class>
   <extends>QWidget</extends>
   <header>okno.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
